pkg/controllers/binding/bind_controller_utils.go

```diff
@@ -303,3 +310,239 @@ func calculateReplicas(c client.Client, policy *v1alpha1.ReplicaSchedulingPolicy
 
 	return desireReplicaInfos, nil
 }
+
```

Comment:
I think we can name this file to `util.go`, because `bind_controller_utils.go` sounds like it's for `binding_controller.go`.

Reply:
ok